lib/brcobranca/boleto/banco_brasil.rb

```diff
@@ -52,14 +52,14 @@ def carteira=(valor)
       #
       # @return [String] 1 caracteres numéricos.
       def banco_dv
-        banco.modulo11_9to2_10_como_x
+        banco.modulo11(mapeamento: {10 => 'X'})
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.